### ISSUE TITLE
first epoch: Error in CuDNN: CUDNN_STATUS_BAD_PARAM (cudnnSetFilterNdDescriptor)

### ISSUE THREAD

**Issue by vjcitn** [eladhoffer/ConvNet-torch#2]
ubuntu@ip-10-81-234-93:~/ConvNet-torch$ !th
th Main.lua -dataset Cifar10 -network Cifar10_Model -LR 0.1
[program started on Sun May 22 22:41:56 2016]  
[command line arguments]  
LR  0.1 
batchSize   128 
network ./Models/Cifar10_Model  
save    /home/ubuntu/ConvNet-torch/Results/SunMay2222:41:542016 
modelsFolder    ./Models/  
devid   1  
augment false  
load  
nGPU    1  
preProcDir  /home/ubuntu/ConvNet-torch/PreProcData/Cifar10  
format  rgb 
whiten  false  
dataset Cifar10 
type    cuda  
momentum    0.9 
visualize   1  
LRDecay 0  
threads 8  
optimization    sgd 
constBatchSize  false  
weightDecay 0.0001  
epoch   -1  
normalization   simple  
[----------------------]  
==> Network 
nn.Sequential {
  [input -> (1) -> (2) -> (3) -> (4) -> (5) -> (6) -> (7) -> (8) -> (9) -> (10) -> (11) -> (12) -> (13) -> (14) -> (15) -> (16) -> (17) -> (18) -> output](1): cudnn.SpatialConvolution(3 -> 64, 5x5)
  (2): cudnn.ReLU
  (3): cudnn.SpatialMaxPooling(2x2, 2,2)
  (4): nn.Dropout(0.500000)
  (5): cudnn.SpatialConvolution(64 -> 128, 3x3)
  (6): cudnn.ReLU
  (7): cudnn.SpatialMaxPooling(2x2, 2,2)
  (8): nn.Dropout(0.500000)
  (9): cudnn.SpatialConvolution(128 -> 256, 3x3)
  (10): cudnn.ReLU
  (11): cudnn.SpatialMaxPooling(2x2, 2,2)
  (12): nn.Dropout(0.500000)
  (13): cudnn.SpatialConvolution(256 -> 128, 2x2)
  (14): cudnn.ReLU
  (15): nn.View(128)
  (16): nn.Dropout(0.500000)
  (17): nn.Linear(128 -> 10)
  (18): cudnn.LogSoftMax
}
==>506378 Parameters  
==> Loss  
nn.ClassNLLCriterion

==> Starting Training
Epoch 1 
/home/ubuntu/torch/install/bin/luajit: /home/ubuntu/torch/install/share/lua/5.1/nn/Container.lua:67: 
In 1 module of nn.Sequential:
/home/ubuntu/torch/install/share/lua/5.1/cudnn/init.lua:58: Error in CuDNN: CUDNN_STATUS_BAD_PARAM (cudnnSetFilterNdDescriptor)
stack traceback:
    [C]: in function 'error'
    /home/ubuntu/torch/install/share/lua/5.1/cudnn/init.lua:58: in function 'errcheck'
    ...torch/install/share/lua/5.1/cudnn/SpatialConvolution.lua:45: in function 'resetWeightDescriptors'
    ...torch/install/share/lua/5.1/cudnn/SpatialConvolution.lua:358: in function <...torch/install/share/lua/5.1/cudnn/SpatialConvolution.lua:357>
    [C]: in function 'xpcall'
    /home/ubuntu/torch/install/share/lua/5.1/nn/Container.lua:63: in function 'rethrowErrors'
    /home/ubuntu/torch/install/share/lua/5.1/nn/Sequential.lua:44: in function 'forward'
    Main.lua:206: in function 'Train'
    Main.lua:257: in main chunk
    [C]: in function 'dofile'
    ...untu/torch/install/lib/luarocks/rocks/trepl/scm-1/bin/th:145: in main chunk
    [C]: at 0x00406670

WARNING: If you see a stack trace below, it doesn't point to the place where this error occured. Please use only the one above.
stack traceback:
    [C]: in function 'error'
    /home/ubuntu/torch/install/share/lua/5.1/nn/Container.lua:67: in function 'rethrowErrors'
    /home/ubuntu/torch/install/share/lua/5.1/nn/Sequential.lua:44: in function 'forward'
    Main.lua:206: in function 'Train'
    Main.lua:257: in main chunk
    [C]: in function 'dofile'
    ...untu/torch/install/lib/luarocks/rocks/trepl/scm-1/bin/th:145: in main chunk
    [C]: at 0x00406670
